public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -117,28 +117,30 @@ define(function(require) {
   }
 
   function save(context) {
-    context.saveButton.text("{{ renameProjectSavingIndicator }}");
-
-    persist.call(context, context.titleBar.val(), function(err) {
-      if(err) {
-        console.error("[Bramble] Failed to rename the project with ", err);
-        return;
-      }
+    // Do not apply save if project title remains the same
+    if (context.titleBar.val() === Project.getTitle()) {
+      editingComplete(context);
+    } else {
+      context.saveButton.text("{{ renameProjectSavingIndicator }}");
```

Comment:
Thanks for the feedback @Pomax!  

I've made the changes in recent commit.  Regarding the template string, all I know is that they get replaced by the values defined in messages.json